src/routes.js

```diff
@@ -19,7 +21,8 @@ export default () => (
         <Route exact path="/signin" component={Signin} />
         <Route path="/signup" component={SignUp} />
         <Route exact path="/login" component={SocialAuth} />
-
+        <Route exact path="/reset/password" component={ForgotPasssword} />
```

Comment:
Can you move this in the Route file, please?

Reply:
I'm guessing if you actually updated develop recently you should have a file under `src/utils/routes.js` that can contain all the routes and you can just call them by `ROUTES.home` for `/`